2021-11-08 12:59:48.199 -03 [31723] LOG:  starting PostgreSQL 13.4 on x86_64-apple-darwin19.6.0, compiled by Apple clang version 11.0.3 (clang-1103.0.32.62), 64-bit
2021-11-08 12:59:48.201 -03 [31723] LOG:  listening on IPv6 address "::1", port 5432
2021-11-08 12:59:48.201 -03 [31723] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2021-11-08 12:59:48.202 -03 [31723] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2021-11-08 12:59:48.205 -03 [31724] LOG:  database system was shut down at 2021-11-08 12:59:48 -03
2021-11-08 12:59:48.208 -03 [31723] LOG:  database system is ready to accept connections
2021-11-08 13:00:01.909 -03 [31810] FATAL:  database "keystone-6-example" does not exist
2021-11-08 13:00:02.041 -03 [31815] ERROR:  relation "_prisma_migrations" does not exist at character 126
2021-11-08 13:00:02.041 -03 [31815] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2021-11-08 13:08:09.679 -03 [32801] FATAL:  database "unionchurch" does not exist
2021-11-08 13:08:09.845 -03 [32814] ERROR:  relation "_prisma_migrations" does not exist at character 126
2021-11-08 13:08:09.845 -03 [32814] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2021-11-08 15:06:35.334 -03 [48437] LOG:  starting PostgreSQL 13.4 on x86_64-apple-darwin19.6.0, compiled by Apple clang version 11.0.3 (clang-1103.0.32.62), 64-bit
2021-11-08 15:06:35.336 -03 [48437] LOG:  listening on IPv6 address "::1", port 5432
2021-11-08 15:06:35.336 -03 [48437] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2021-11-08 15:06:35.337 -03 [48437] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2021-11-08 15:06:35.344 -03 [48438] LOG:  database system was interrupted; last known up at 2021-11-08 13:13:18 -03
2021-11-08 15:06:35.452 -03 [48438] LOG:  database system was not properly shut down; automatic recovery in progress
2021-11-08 15:06:35.456 -03 [48438] LOG:  redo starts at 0/1712E28
2021-11-08 15:06:35.456 -03 [48438] LOG:  invalid record length at 0/1712F10: wanted 24, got 0
2021-11-08 15:06:35.456 -03 [48438] LOG:  redo done at 0/1712ED8
2021-11-08 15:06:35.464 -03 [48437] LOG:  database system is ready to accept connections
2021-11-08 15:06:40.173 -03 [48450] LOG:  could not receive data from client: Connection reset by peer
2021-11-08 17:56:22.222 -03 [67369] LOG:  could not receive data from client: Connection reset by peer
2021-11-08 17:56:55.759 -03 [67460] LOG:  could not receive data from client: Connection reset by peer
